pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -12,6 +12,7 @@ description: AP Pipeline with synthetic/fake sources. Templates are inputs.
 imports:
   - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
   - location: $ANALYSIS_TOOLS_DIR/pipelines/apDetectorVisitQualityCore.yaml
+  - location: $ANALYSIS_TOOLS_DIR/pipelines/apFakesMetrics.yaml
```

Comment:
Also add the new tasks to the appropriate subsets.  That includes `apPipe` and `injected_apPipe`. After you rebase, they should also go in the new `afterburner` subset.